smoke/IotEdgeQuickstart/Quickstart.cs

```diff
@@ -62,7 +68,11 @@ public async Task RunAsync()
                 {
                     await this.StartBootstrapper();
                     await this.VerifyEdgeAgentIsRunning();
-                    await this.VerifyEdgeAgentIsConnectedToIotHub();
+                    if (!this.bypassEdgeInstallation)
+                    {
+                        await this.VerifyEdgeAgentIsConnectedToIotHub();
```

Comment:
we should still verify edge agent is connected to iot hub even iotedge installation is not required. right? if not, why?

Reply:
Agree.